src/components/Content/ContentMining.vue

```diff
@@ -52,7 +52,19 @@
             <b>Ore</b>.
           </span>
           <span>
-            Of course, you're probably going to want to build some experience up before you go after some of the
+            Beginners start by pounding 
+            <img
+              class="mx--2"

```

Comment:
```suggestion
              class="mx--0"

```

These classes are bad and I hate them, but `mx--2` means `margin x negative 2`. You'll only want a number there when the sprite has some padding on the side we want to get rid of, otherwise it clips in like this (compare vein to sand)
![image](https://user-images.githubusercontent.com/16600313/87172973-14a46f80-c2a3-11ea-83a4-204b0d4edbf7.png)